assets/js/greeter/Greeter.tsx

```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from 'react'
 
 interface Props {
-	id: number
```

Comment:
let's keep the 2 spaces, either way the nesting will be insane

Reply:
I'm using 2 spaces tab for indentation, same as `index.ts`

Reply:
Then why it's triggering those changes?

Reply:
Yes, I'm keeping 2 spaces, I think is an issue with GH preview